package-lock.json

```diff
@@ -8332,9 +8332,9 @@
       }
     },
     "node_modules/hugo-bin": {
-      "version": "0.125.0",
-      "resolved": "https://registry.npmjs.org/hugo-bin/-/hugo-bin-0.125.0.tgz",
-      "integrity": "sha512-cTvHzQdK04KrBaW3jPdd/JAQXCOdXT1GiPHaMxe0YtLTVCV0ElgxkvXn6eQJUDl4QqM8+IlsFg/b2ofz6XS//A==",
+      "version": "0.125.1",
```

Comment:
While not strictly needed would be good to keep the `package.json` in sync.

Reply:
Oh I can fire another PR in. I just didn't know how to do that. I kinda figured npm woulda done whatever was needed.

Reply:
Dependabot should end up doing that in the next update.

Reply:
Okay that's fair, but any idea for next time?

Reply:
I think an update before doing the audit fix.